Nodejs/Product/TestAdapter/TestDiscoverer.cs

```diff
@@ -119,7 +145,7 @@ private void DiscoverTests(Dictionary<string, List<TestFileEntry>> testItems, MS
 
             if (!File.Exists(nodeExePath))
             {
-                logger.SendMessage(TestMessageLevel.Error, string.Format(CultureInfo.CurrentCulture, "Node.exe was not found.  Please install Node.js before running tests."));
+                logger.SendMessage(TestMessageLevel.Error, "Node.exe was not found. Please install Node.js before running tests.");
```

Comment:
Where do all these (unlocalized) error messages appear?

Reply:
They are shown in the Test Log... none of the output in that assembly is currently localized, I'll do that in a separate PR